logp/configure/logging.go

```diff
@@ -79,6 +79,7 @@ func LoggingWithOutputs(beatName string, cfg *config.C, outputs ...zapcore.Core)
 	return logp.ConfigureWithOutputs(config, outputs...)
 }
 
+// Deprecated (moving towards localized loggers): Use logp.LoggingWithTypedOutputsLocal.
 // LoggingWithTypedOutputs applies some defaults then calls ConfigureWithTypedOutputs
```

Comment:
In order for godoc to report the API as deprecated it needs to be written in a specific way.

See https://go.dev/wiki/Deprecated.

```suggestion
// LoggingWithTypedOutputs applies some defaults then calls ConfigureWithTypedOutputs
//
// Deprecated: Moving towards localized loggers. Use logp.LoggingWithTypedOutputsLocal.
```